package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web3-plugin-eip4337",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
I'll suggest to start 0.1.0 as its first release will be dev / alpha release. and after that in some next PR make it 1.0.0 when its prod ready.